### PR TITLE
Added js functionality and a button to the modal

### PR DESCRIPTION
**Pull request by eliashedlund**

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -50,8 +50,19 @@ gulp.task('styleguide-sass', function() {
     .pipe(notify({message: 'Sass task complete'}));
 });
 
+gulp.task('styleguide-assets', function() {
+  var files = {
+    'bower_components/bootstrap-sass-twbs/assets/javascripts/bootstrap.min.js': 'public/javascripts',
```
Comment (mnd-hound) [inline gulpfile.js]: Line is too long.

File: gulpfile.js [hunk 2]
```diff
@@ -50,8 +50,19 @@ gulp.task('styleguide-sass', function() {
...
+    'bower_components/jquery/dist/jquery.min.js': 'public/javascripts'
+  }
```
Comment (mnd-hound) [inline gulpfile.js]: Missing semicolon.

File: gulpfile.js [hunk 3]
```diff
@@ -50,8 +50,19 @@ gulp.task('styleguide-sass', function() {
...
+  for (file in files) {
```
Comment (mnd-hound) [inline gulpfile.js]: Creating global 'for' variable. Should be 'for (var file ...'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.

File: gulpfile.js [hunk 4]
```diff
@@ -50,8 +50,19 @@ gulp.task('styleguide-sass', function() {
...
+    gulp.src(file)
+      .pipe(gulp.dest(files[file]));
+  }
+});
+
 // Compile style-guide
-gulp.task('hologram', ['clean', 'sass', 'styleguide-sass'], function() {
+gulp.task('hologram', ['clean', 'sass', 'styleguide-sass', 'styleguide-assets'], function() {
```
Comment (mnd-hound) [inline gulpfile.js]: Line is too long.

File: gulpfile.js [hunk 5]
```diff
@@ -50,8 +50,19 @@ gulp.task('styleguide-sass', function() {
     .pipe(notify({message: 'Sass task complete'}));
 });
 
+gulp.task('styleguide-assets', function() {
+  var files = {
+    'bower_components/bootstrap-sass-twbs/assets/javascripts/bootstrap.min.js': 'public/javascripts',
+    'bower_components/jquery/dist/jquery.min.js': 'public/javascripts'
+  }
+  for (file in files) {
+    gulp.src(file)
```
Comment (mnd-hound) [inline gulpfile.js]: 'file' is not defined.

File: gulpfile.js [hunk 6]
```diff
@@ -50,8 +50,19 @@ gulp.task('styleguide-sass', function() {
...
+      .pipe(gulp.dest(files[file]));
```
Comment (mnd-hound) [inline gulpfile.js]: 'file' is not defined.

**Comment by eliashedlund** [comment]
Can we ignore "line is too long"

**Comment by ewal** [comment]
Nice, but it seems like bootstrap.min is missing from the public folder.

**Comment by ewal** [comment]
All good. Needed to update the bower component bootstrap-sass-twbs in order to get the bootstrap.min file.

**Pull request merged**